src/aws-cpp-sdk-core/include/aws/core/AmazonWebServiceRequest.h

```diff
@@ -203,6 +203,11 @@ namespace Aws
 
         virtual Aws::Client::CompressionAlgorithm
         GetSelectedCompressionAlgorithm(Aws::Client::RequestCompressionConfig) const { return Aws::Client::CompressionAlgorithm::NONE; }
+        /*
```

Comment:
please move it up right after `IsEventStreamRequest`
please also use a consistent docstring style of this file

```
        /**
         * Defaults to false, if this is set to true in derived class, the operation using this request will return an event stream response.
         */
        inline virtual bool HasEventStreamResponse() const { return false; }
```